hammer/technology/sky130/sky130.tech.json

```diff
@@ -9,10 +9,6 @@
     {
       "id": "$SKY130A",
       "path": "technology.sky130.sky130A"
-    },
-    {
-      "id": "$SKY130_CDS",
```

Comment:
Why did you remove all this stuff?

Reply:
was it manually inserted? i just reran gen-tech-json with my changes and overwrote

Reply:
can manually revert if so